### PR TITLE
KG - Custom Dashboard Protocol Sorting

### PR DESCRIPTION
**Pull request by kyle-glick**

### CODE REVIEW

**Comment by kyle-glick** [comment]
@ myself: Remove sorted_by from protocol_filter model / migration

File: app/controllers/dashboard/protocols_controller.rb [hunk 1]
```diff
@@ -233,6 +234,17 @@ def find_protocol
...
   end
 
+  def setup_sorting_variables
+    # Set filterrific params for sorting logic, store sorted by to re-apply styling
+    @filterrific_params = params[:filterrific] ? params[:filterrific].except(:sorted_by) : @default_filter_params
+    @sorted_by          = params[:filterrific][:sorted_by] if params[:filterrific]
+    @sort_name          = @sorted_by.split(' ')[0] if @sorted_by
+    @sort_order         = @sorted_by.split(' ')[1] if @sorted_by
+    @new_sort_order     = (@sort_order == 'asc' ? 'desc' : 'asc') if @sort_order
+    puts "!"*50
```
Comment (jayhardee9) [inline app/controllers/dashboard/protocols_controller.rb]: Maybe you overlooked these puts's

**Comment by wtholt** [comment]
Looks good to me.

File: app/models/protocol.rb [hunk 2]
```diff
@@ -227,8 +224,8 @@ def self.options_for_sorted_by
       ['ID (9-0)', 'id_desc'],
```
Comment (jleonardw9) [inline app/models/protocol.rb]: please remove

**Pull request merged**